handling-forms/problem-set-forms-and-apis.md

```diff
@@ -31,6 +55,48 @@ The web app should also display:
 - The latitudinal coordinate of the search result
 - The longitudinal coordinate of the search result
 
+## Resources
+
+Scaffold code is available to help you get started quickly.
+
+- [Forms and APIs Problem Set](https://github.com/AdaGold/problem-set-forms-and-apis)
+
+After forking and cloning the project, run `npm install` to install the necessary dependencies. The development server is started with `npm run dev`.
+
+The scaffold code removes a lot of the default Vite template code and simplifies the applied styles. The `App` component renders an empty body.
+
+A `.env.sample` file is provided that can be used to store the API key. Make a copy of the file and name it `.env`. Then update the LocationIQ key in that file. The `.env` file is already added to the `.gitignore` file, so it will not be committed to the repository. We should still avoid attempting to deploy this to any public location, as the API key will still be visible in the source code.
+
+## Implement
+
+### !callout-error
+
+## Use Caution When Committing Your Work
+
+This is a front end application that uses an API key. Be careful not to commit the API key to a public repository (such as by hardcoding it into a JavaScript file).
+
+<br />
+
+*Do not deploy this application to a public server.* Even if we use React's version of a `.env` file, the API key will be visible to anyone who views the source code using the browser development tools.
+
+<br />
+
+If we ever accidentally commit an API key, we should immediately revoke the key and generate a new one.
+
+### !end-callout
+
+### Wave 1: Implement the Goal User Story
+
+Build and implement the described interface and functionality. We will need to make use of `axios` to make API calls. Note that the API call is to be performed in response to a user action, not on page load.
+
+While it is possible to implement the entire project in a single component, consider breaking the project into smaller components. For example, we could have a `SearchForm` component that handles the form input, and a `SearchResult` component that displays the results. When splitting up the project, consider the following:
+- What data must be tracked by the app?
+- Where should the single source of truth for the data be?
+- What props are needed to pass data between components?
+- How should the data be updated?
+- How can we ensure that data is updated after the API call is complete?
+- What logic must reside in a component, and what logic can be relocated to other locations?
```

Comment:
I wanted to give the reader some quesitons to use to help think about their approach. For folks new to React, this problem set is actually quite large, so I was hoping to avoid blank canvas syndrome.

Maybe I should include some collapsed answers to the questions? That could help with the intent concern, and give folks more direction after they've had a chance to think on their own?